addons.py

```diff
@@ -58,7 +58,8 @@
     "plugin.video.titulkomet.cz",
     "weather.shmu.pocasie",
     "skin.estuary.stream-cinema",
-    "plugin.video.sl"
+    "plugin.video.sl",
+	"plugin.video.mall.tv"
```

Comment:
Zachovat sposob odsadzovania 4mi medzerami:
```suggestion
    "plugin.video.mall.tv"
```

Reply:
Děkuji za upozornění. Zřejmě chyba editoru.